test/avocado/checklogin-basic.py

```diff
@@ -95,9 +95,9 @@ def login(user, password):
         b.wait_text("#account-user-name", "admin")
 
         c.allow_journal_messages("Returning error-response ... with reason .*",
-                                 "pam_unix\(cockpit:auth\): authentication failure; .*",
-                                 "pam_unix\(cockpit:auth\): check pass; user unknown",
-                                 "pam_succeed_if\(cockpit:auth\): requirement .* not met by user .*")
+                                 r"pam_unix\(cockpit:auth\): authentication failure; .*",
```

Comment:
There is really no practical difference, as backslash escapes in "normal" strings aren't magic in front of anything other than `\` and `"`. But indeed it looks a bit nicer, and many tutorials suggest using raw strings for regexps just to be on the safe side.

Reply:
Yes, but,it's just for good practice. :)